pom.xml

```diff
@@ -1620,6 +1624,67 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>enforce-jdk11</id>
+            <activation>
+                <jdk>11</jdk>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-enforcer-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>enforce-versions</id>
+                                <goals>
+                                    <goal>enforce</goal>
+                                </goals>
+                                <configuration>
+                                    <rules>
+                                        <requireJavaVersion>
+                                            <version>[11.3,)</version>
```

Comment:
As far as I know, this would mean `11.3` or **ANY** version higher than 11.3 which would include jdk12, jdk13, jdk14, etc? This rule and the `jdk8` would conflict.

Reply:
The jdk version could be upper bounded by jdk12

Reply:
> The jdk version could be upper bounded by jdk12

As it's in a profile which is activated only, if the JDK is 11, this looks correct to me. Could you please test is with JDK 11 and 12 and confirm the behaviour and output? Thanks! :)

Reply:
@carlspring Here is the output for JDK 11 and JDK 12
![image](https://user-images.githubusercontent.com/46301867/95768964-f9fbe100-0c6b-11eb-866d-3367aa7659d3.png)